os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -791,6 +791,58 @@ trble_result_e rtw_ble_client_operation_enable_notification(trble_operation_hand
     return rtw_ble_client_operation_write(handle, &in_data);
 }
 
+trble_result_e rtw_ble_client_operation_enable_indication(trble_operation_handle* handle)
+{
+    if (handle == NULL)
+    {
+        return TRBLE_FAIL;
+    }
+
+    if(!le_get_active_link_num())
+    {
+        debug_print("No active connection \n");
+        return TRBLE_FAIL;
+    }
+
+    if (client_init_parm == NULL || client_init_parm->trble_operation_notification_cb == NULL)
+    {
+        return TRBLE_FAIL;
```

Comment:
I think it is better to print the log why it fails.

Reply:
I think it's not necessary to print the fail cause here.
The API returned TRBLE_FAIL if fail causes occurred.
Only when the fail cause is invalid pointer, there is no debug log printed; For other fail cases, corresponding debug log will be printed. So I think current code is enough for debugging.